build-system/tasks/extension-helpers.js

```diff
@@ -30,16 +30,16 @@ const {
   compileJs,
   compileJsWithEsbuild,
   mkdirSync,
+  doBuildJs,
 } = require('./helpers');
 const {analyticsVendorConfigs} = require('./analytics-vendor-configs');
 const {compileJison} = require('./compile-jison');
-const {endBuildStep, watchDebounceDelay, doBuildJs} = require('./helpers');
+const {endBuildStep, watchDebounceDelay} = require('./helpers');
```

Comment:
We're importing from `helpers.js` on this line and on lines 28-34. Can we consolidate into one list?